### ISSUE TITLE
Aligning toggles to the right

### ISSUE THREAD

**Issue by takase1121** [lite-xl/lite-xl-widgets#5]
The current alignment for Toggle and its label is horrible.

![image](https://user-images.githubusercontent.com/20792268/217533252-2eca6749-61a8-499f-988b-db3fc98a9b2a.png)

In contrast, this is how it looks like in Material design:
![image](https://user-images.githubusercontent.com/20792268/217533320-39ba7fb6-4221-4105-ba10-5c28ecaebc77.png)

This makes it less jarring to use.

On another note, radio buttons are usually left-aligned so that the buttons are kept in a straight line:

![image](https://user-images.githubusercontent.com/20792268/217533522-b3430e86-3cf6-479a-aba2-ddbc91542041.png)

**Comment by jgmdev** [comment]
Good point, as you are pointing out they can be moved to left side of label like radio buttons. Also, maybe add a method to decide the toggle position, left or right and when placed right place it at total_size_of_toggle_including_label - size_of_toggle_box.,
